protocol/0028-GOVE-governance.md

```diff
@@ -131,6 +131,27 @@ If a proposal is created and later a different outcome is preferred by network p
 
 Which of these makes most sense will depend on the type of change, the timing of the events, and how the rest of the community votes for the initial proposal.
 
+## Cancelling a proposal
+
+A proposal can be cancelled under certain conditions by, and only by, the party which created the proposal. To avoid situations where the community may have acted upon proposals they "expected" to pass, a party can only cancel their proposal before a certain "cancellation threshold" is reached.
+
+This cancellation threshold is a fixed point before the proposal closing time. The duration of the period in which a proposal cannot be cancelled is defined by the network parameter `governance.proposal.cancellationThreshold` (a duration string defaulting to `24h0m0s`).
+
+i.e. Given the `governance.proposal.cancellationThreshold` is `24h0m0s` a party will be able to cancel their proposal up until 1 day before the proposal closing time.
+
+### Mechanics
+
+A party can cancel their proposal by simply voting against it with any vote value.
+
+- If the current time is before the cancellation threshold, the proposal is instantly rejected.
+- If the current time is after the cancellation threshold, the vote is treated normally as a vote against a proposal.
+
+Note the following additional details:
+
+- a cancelling vote against a proposal must pass all typical vote checks and spam protections, i.e. who party who removes stake after creating a proposal and no longer meets the requirement will not be able to cancel their proposal.
+
+- if at the time of creating a proposal the closing time is within the period defined by `governance.proposal.cancellationThreshold`, the proposal will still be valid and accepted by the network but the party will not have an opportunity to cancel the proposal.
```

Comment:
We should mention that their vote will still be counted as (changed to) a vote against even if it is too late to canel the proposal

Reply:
Yeah I agree, hopefully `line 147` makes this point already though?